lib/functions/dts-process.js

```diff
@@ -0,0 +1,71 @@
+const logger = require('../helpers/logging')
+const pg = require('../helpers/db')
+const invokeLambda = require('../helpers/invoke-lambda')
+const { deleteStation, getRloiIds, getStationData, validateStationData } = require('../helpers/imtd-api')
+
+async function insertStation (stationDataArray) {
+  try {
+    await pg.transaction(async trx => {
+      await Promise.all(stationDataArray.map(async (stationData) => {
+        const stationID = stationData.station_id
+        await trx('station_display_time_series').where({ station_id: stationID }).delete()
+        await trx('station_display_time_series').insert(stationData)
+        logger.info(`Processed displayTimeSeries for RLOI id ${stationID}`)
+      }))
+    })
+  } catch (error) {
+    logger.error('Database error processing stationData', error)
+    throw error
+  }
+}
+
+async function getData (stationId) {
+  try {
+    const stationData = await getStationData(stationId)
+    if (stationData.length === 0) {
+      (console.log('Deleting station: ', stationId))
```

Comment:
don't think need the extra set of parentheses around the console.log, could just be `console.log('Deleting station: ', stationId)` , doesn't matter functionally though, still do what is intended.